resources/js/components/Kiosk.vue

```diff
@@ -471,11 +481,11 @@ export default {
                         });
                     } else if (error.response.status === 422 && error.response.data.error.includes('space') ) {
                         let msg = error.response.data.error
-                        msg += " You must set your default space at " + window.location.hostname
+                        msg += "<b> You must set your default space at " + window.location.hostname + " before punching in.</b>"
```

Comment:
In my pedantic opinion I would use `<strong>` instead of `<b>` but it's also moot here

Reply:
leaving it for now